cli/medperf/tests/commands/test_list.py

```diff
@@ -0,0 +1,82 @@
+from medperf.commands.list import EntityList
+import medperf.commands.list as list_module
+from medperf.exceptions import InvalidArgumentError
+import pytest
+from medperf.entities.interface import Entity
+
+
+def generate_display_dicts():
+    """Mocks a list of dicts to be returned by Entity.display_dict
+    """
+    return [{f"k{j}": f"v{i}{j}" for j in range(4)} for i in range(3)]
+
+
+@pytest.fixture()
+def setup(request, mocker, ui):
+    # system inputs
+    display_dicts = request.param["display_dicts"]
+
+    # mocks
+    entity_object = mocker.create_autospec(spec=Entity)
+    mocker.patch.object(entity_object, "display_dict", side_effect=display_dicts)
+
+    # spies
+    generated_entities = [entity_object for _ in display_dicts]
+    all_spy = mocker.patch(
+        "medperf.entities.interface.Entity.all", return_value=generated_entities
+    )
+    ui_spy = mocker.patch.object(ui, "print")
+    tabulate_spy = mocker.spy(list_module, "tabulate")
+
+    system_inputs = {"display_dicts": display_dicts}
```

Comment:
Putting on my "dumb reader" hat on 😆. This variable `system_inputs` isn't clear from a first read. I personally don't know exactly what this means in the context of the tests.

Reply:
@hasan7n thinking about this, do you think using `state_variables` would be a more intuitive name? If so it would be nice to change that name in all the tests that use it so far

Reply:
sounds perfect!